spec/index.bs

```diff
@@ -1589,6 +1589,11 @@ applications should generally request the "worst" limits that work for their con
         {{GPURenderPassDescriptor}}.{{GPURenderPassDescriptor/colorAttachments}},
         and {{GPURenderPassLayout}}.{{GPURenderPassLayout/colorFormats}}.
 
+    <tr><td><dfn>maxTotalColorAttachmentDepth</dfn>
```

Comment:
"Depth" is slightly confusing here because we aren't able to say the less ambiguous "bit depth".
I might call this... `maxColorAttachmentBytesPerFragment`? `PerSample`?

Reply:
well, it's not per fragment, and it's not per sample. I think the term-of-art for this is, surprisingly, per pixel. `maxColorAttachmentBytesPerPixel` it is.